charts/agent-group/values.schema.json

```diff
@@ -0,0 +1,257 @@
+      "properties": {
+        "registerjson": {
+          "oneOf": [
+            {
+              "additionalProperties": false,
+              "type": "object"
+            },
+            {
+              "additionalProperties": false,
+              "required": [
+                "cloudUrl",
+                "username",
+                "password",
+                "agentGroupId"
+              ],
+              "type": "object",
+              "properties": {
+                "username": {
+                  "type": "string",
+                  "description": "Encrypted username of the Agent Installer user (encrypted using JitterbitUtils)"
+                },
+                "agentNamePrefix": {
+                  "type": "string",
+                  "description": "Prefix for each auto-registered agent's name"
+                },
+                "deregisterAgentOnDrainstop": {
+                  "type": "boolean",
+                  "description": "Performs agent deregistration on drainstop/JVM shutdown (defaults to false)"
+                },
+                "retryIntervalSeconds": {
+                  "minimum": 5,
+                  "type": "integer",
+                  "description": "Number of seconds it will wait before retrying. This number doubles every retry to a max of 10 minutes (defaults to 5, valid range 5-600)",
+                  "maximum": 600
+                },
+                "agentGroupId": {
+                  "type": "number",
+                  "description": "Agent group Id from Management Console (hover over an agent group to see its Id)"
+                },
+                "retryCount": {
+                  "minimum": 0,
+                  "type": "integer",
+                  "description": "Number of retries if agent is having issues making the call to Harmony cloud for registration (defaults to 10, valid range is 0-300)",
+                  "maximum": 300
+                },
+                "password": {
+                  "type": "string",
+                  "description": "Encrypted password of the Agent Installer user (encrypted using JitterbitUtils)"
+                },
+                "cloudUrl": {
+                  "type": "string",
+                  "description": "Base URL of the Jitterbit Harmony Cloud"
+                }
+              }
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "description": "agent group's register.json file"
+        },
+        "jitterbitconf": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Agent group's jitterbit.conf file"
+        }
+      }
+    },
+    "affinity": {
+      "description": "Pod affinity and anti-affinity definitions",
+      "$ref": "#/definitions/affinity"
+    },
+    "env": {
+      "description": "List of extra environment variables that may be specified in the container",
```

Comment:
"description": "List of additional environment variables that may be specified in the container",